tests/weather_test.go

```diff
@@ -22,6 +24,14 @@ func init() {
 
 // TestGet is a sample to run an endpoint test
 func TestWeatherGet(t *testing.T) {
+	db := models.MySQLWeatherDb{
```

Comment:
Look for to provoke some errors in the external api and build some test case  for handler these errors